src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -886,6 +916,72 @@ GrpAb: Z
     return codomain(map_from_torusinvariant_cartier_divisor_group_to_picard_group(v))
 end
 
+@doc raw"""
+    map_from_picard_group_to_class_group(v::AbstractNormalToricVariety)
+
+Return the embedding of the Picard group into the class group of an abstract normal toric variety `v`.
+
+# Examples
+```jldoctest
+julia> p2 = projective_space(NormalToricVariety, 2)
+Normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> map_from_picard_group_to_class_group(p2)
+Map with following data
+Domain:
+=======
+Abelian group with structure: Z
+Codomain:
+=========
+Abelian group with structure: Z
+```
+"""
+@attr GrpAbFinGenMap function map_from_picard_group_to_class_group(v::AbstractNormalToricVariety)
+    f = image(map_from_torusinvariant_cartier_divisor_group_to_class_group(v))[2]
+    g = snf(domain(f))[2] * f
+    return hom(picard_group(v), class_group(v), matrix(g))
+end
+
+
+############################
+# Gorenstein and Picard Index
+############################
+
+@doc raw"""
+    gorenstein_index(v::AbstractNormalToricVariety)
+
+Return the Gorenstein index of an abstact normal toric variety `v`.
+
+# Examples
+```jldoctest
+julia> gorenstein_index(weighted_projective_space(NormalToricVariety, [2,3,5]))
+3
+```
+"""
+@attr ZZRingElem function gorenstein_index(v::AbstractNormalToricVariety)
+    @req is_q_gorenstein(v) "gorenstein index can only be computed for Q-gorenstein varieties"
+    c = divisor_class(anticanonical_divisor_class(v))
+    f = cokernel(map_from_picard_group_to_class_group(v))[2]
+    order(f(c))
+end
+
+
+@doc raw"""
+    picard_index(v::AbstractNormalToricVariety)
+
+Return the index of the Picard group in the class group of an abstact normal toric variety `v`.
+
```

Comment:
Similar to the above comment on the Gorenstein index - consider adding a few explaining words and/or a reference (if not introduced in CLS11 - I could not yet find it in there).

Reply:
Thank you explaining this index in the doc string. This is definitely sufficient to know what this index is.

I agree with @lkastner that it would be nice to have a reference and if so, link it here. However, if you really cannot find anything, I think this should not be a leg breaker. Unless @lkastner objects, I am good to go without this reference.

Reply:
I have added a reference in the docstring of `picard_index`, let me know what you think. It doesn't really go into detail about it, but it is a reference where the Picard index was used in a classification.